src/js/common/schemaform/widgets/index.js

```diff
@@ -3,13 +3,17 @@ import SelectWidget from './SelectWidget';
 import DateWidget from './DateWidget';
 import EmailWidget from './EmailWidget';
 import RadioWidget from './RadioWidget';
+import CheckboxWidget from './CheckboxWidget';
+import YesNoWidget from './YesNoWidget';
 
 const widgets = {
   TextWidget,
   SelectWidget,
   DateWidget,
   EmailWidget,
-  RadioWidget
+  RadioWidget,
+  CheckboxWidget,
+  yesNo: YesNoWidget
```

Comment:
I explained it some above, but this is mainly to make the form config more consistent looking. If I use YesNoWidget, then in the config I'll have lines like `'ui:widget': 'YesNoWidget'`, whereas in other places you would do things like `'ui:widget': 'radio'` to get `RadioWidget`.